File: BOJ_2441/BOJ_2441.java
import java.io.*;

class Main{
        public static void main(String[] args)
                        throws IOException{
                        BufferedReader BR = new BufferedReader(new InputStreamReader(System.in));
                        BufferedWriter BW = new BufferedWriter(new OutputStreamWriter(System.out));
                        StringBuilder SB = new StringBuilder();
                        int N = Integer.parseInt(BR.readLine());
                        for(int i=0; i<N; i++){
                                for(int j=0; j<i; j++){
                                        SB.append(" ");
                                }
                                for(int k=0; k<N-i; k++){
                                        SB.append("*");
                                }
                                SB.append("\n");
                        }
                        BW.write(SB + "");
                        BW.flush();

        }
}

File: BOJ_2441/BOJ_2441.cs
using System;
namespace asharien{
        class PROGRAM{
                static void Main(){
                        int N = int.Parse(Console.ReadLine());
                        for(int i=0; i<N; i++){
                                for(int j=0; j<i; j++){
                                        Console.Write(" ");
                                }
                                for(int k=0; k<N-i; k++){
                                        Console.Write("*");
                                }
                                Console.WriteLine();
                        }
                }
        }
}
